pymc_marketing/mmm/components/base.py

```diff
@@ -21,13 +21,65 @@
 """
 
 import warnings
+from collections.abc import Generator, MutableMapping, Sequence
 from inspect import signature
+from itertools import product
 from typing import Any
 
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+import numpy.typing as npt
+import pymc as pm
+import xarray as xr
+from pymc.distributions.shape_utils import Dims
 from pytensor import tensor as pt
 
 from pymc_marketing.mmm.utils import _get_distribution_from_dict
 
+Values = Sequence[Any] | npt.NDArray[Any]
+Coords = dict[str, Values]
+
+
+def get_plot_coords(coords: Coords) -> Coords:
+    plot_coord_names = list(coords.keys())[3:]
```

Comment:
Where is this `3` coming from? Any way we can pass a parameter or a name for it?

Reply:
First two coords are "chain", "draw" then third one is the x-axis in the plot which has either name "x" or "time since exposure". Could be better to do: 
```suggestion
    plot_coord_names = list(key for key in coords.keys() if key not in ["chain", "draw", "x", "time since exposure"])
```
Any following coordinates will be plotting as part of the grid

Reply:
Yes, this is clearer for the reader :) 🙏

Reply:
added in